scripts/SetupPremake.py

```diff
@@ -0,0 +1,51 @@
+import sys
+import os
+from pathlib import Path
+
+import Utils
+
+class PremakeConfiguration:
+    premakeVersion = "5.0.0-alpha16"
+    premakeZipUrls = f"https://github.com/premake/premake-core/releases/download/v{premakeVersion}/premake-{premakeVersion}-windows.zip"
+    premakeLicenseUrl = "https://github.com/premake/premake-core/blob/master/LICENSE.txt"
```

Comment:
If you look at the current state of the [`license.txt`](https://github.com/TheCherno/Hazel/blob/b29332e58962fb75e2e61f16e301f124cc88fac4/vendor/premake/bin/LICENSE.txt) file, you'll see that it downloaded the actual github page rather than the txt file. This is easily resolved by changing the `premakeLicenseUrl` to the raw txt file:
```suggestion
    premakeLicenseUrl = "https://raw.githubusercontent.com/premake/premake-core/master/LICENSE.txt"
```

Reply:
I'm sorry, I didn't check the content of the license file, I just tested and see if it actually download something :(